<!--Display for checking the areas defined as cells.
    Pressing the 's' key, the filtered images for each cell are displayed too.
    The information is read from an xml file loaded in the parameter server.
-->

<launch>
  <include file="$(find baxter_tictactoe)/launch/usb_cam_node.launch" />

  <!-- xml file containing the cells of the board -->
<!--   <param name="board_file" textfile="$(find baxter_tictactoe)/resources/ttt_board_lab.xml" />

  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find baxter_tictactoe)/custom_rosconsole.conf"/> -->

  <node name="cells_definition_auto" pkg="baxter_tictactoe" type="cells_definition_auto" respawn="false" output="screen">
    <remap from="/image_in" to="/usb_cam/image_raw"/>
  </node>

<!--   <node name="show_board_cells" pkg="baxter_tictactoe" type="show_board_cells" respawn="false">
    <remap from="/image_in" to="/usb_cam/image_raw"/>        
  </node> -->

  <node name="show_board_cells" pkg="baxter_tictactoe" type="show_board_cells" respawn="false" output="screen">
    <remap from="/image_in" to="/usb_cam/image_raw"/>        
  </node>
 
</launch>
